ansible/install/grafana-dashboards.yml

```diff
@@ -3,7 +3,8 @@
 # Playbook to upload CFME/Miq Grafana Performance Dashboards to Grafana host
 #
 
-- hosts: localhost
+# - hosts: localhost
+- hosts: monitorhost
```

Comment:
I believe changing this will introduce breakage on "All-in-one" monitored appliance or possibly the need to adjust that to work with this.